src/src_python/antares_xpansion/xpansionConfig.py

```diff
@@ -13,7 +16,19 @@ class XpansionConfig():
     # pylint: disable=too-many-instance-attributes
     # pylint: disable=too-few-public-methods
 
-    def __init__(self):
+    def __init__(self, configfile: Path):
+
+        with open(configfile) as file:
+            content = yaml.full_load(file)
+            if content is not None:
+                self.ANTARES = content.get('ANTARES', "antares-solver")
+                self.MERGE_MPS = content.get('MERGE_MPS', "merge_mps")
+                self.BENDERS_MPI = content.get('BENDERS_MPI', "bender_mpi")
+                self.BENDERS_SEQUENTIAL = content.get('BENDERS_SEQUENTIAL', "benders_sequential")
+                self.LP_NAMER = content.get('LP_NAMER', "lp_namer")
```

Comment:
Use `sys.platform` to set default values for Linux/Apple "application" or for Windows "application.exe".

So that default values are relevant for each OS family.

Reply:
I think this will make code less readable and you should have correct values in .yaml because file is generated by cmake.

Reply:
Then remove default values and throw an error if any attribute is not found (future revision)